mslice/plotting/plot_window/cut_plot.py

```diff
@@ -212,18 +218,14 @@ def calc_figure_boundaries(self):
     def xy_config(self):
         return {'x_log': self.x_log, 'y_log': self.y_log, 'x_range': self.x_range, 'y_range': self.y_range}
 
-
-    def _has_errorbars(self):
-        """True current axes has visible errorbars,
-         False if current axes has hidden errorbars"""
+    def _single_line_has_error_bars(self, index):
```

Comment:
Does `index` refer to the index of the line? If so, let's make this clearer in the name.

Reply:
Yes it does, I'll change it